### PR TITLE
Added 3rd and 2nd to last in iteration 4 and some for seasonal_summary

### PR DESCRIPTION
**Pull request by mcat56**
@sasloan  @zacisaacson  @freeheeling

### CODE REVIEW

File: lib/team_stats.rb [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+      end
+    end
+    post_and_reg
+  end
+
+  def generate_win_percentage_by_post_and_reg_per_season(team_id)
+
+
+  end
+
```
Comment (ghost) [inline lib/team_stats.rb:96]: lots of space?

File: lib/team_stats.rb [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+        post_and_reg[@games[id].season][:regular_season] = [id]
+      elsif @games[id].type == "Regular Season" && post_and_reg.has_key?(@games[id].season)
+        post_and_reg[@games[id].season] = {}
+        post_and_reg[@games[id].season][:regular_season] = []
+        post_and_reg[@games[id].season][:regular_season] << id
+      end
+    end
+    post_and_reg
+  end
+
```
Comment (ghost) [inline lib/team_stats.rb:91]: do we need to write the formula for this method? or delete it?

**Review by ghost — APPROVED** [review]
other than the spacing and the empty method it looks good

File: test/season_stats_test.rb [hunk 3]
```diff
@@ -12,50 +12,10 @@ def setup
...
-    skip
-    assert_equal "FC Cincinnati", @stat_tracker.most_tackles("20132014")
-    assert_equal "Seattle Sounders FC", @stat_track.most_tackles("20142015")
-  end
-
-  def test_fewest_tackles
-    skip
-    assert_equal "Atlanta United", @stat_tracker.fewest_tackles("20132014")
-    assert_equal "Orlando City SC", @stat_tracker.fewest_tackles("20142015")
-  end
```
Comment (zacisaacson) [inline test/season_stats_test.rb:59]: Ill recommit the season_stats_test

File: lib/team_stats.rb (hunk 2 above)
Comment (mcat56) [reply lib/team_stats.rb:91]: we will be using it, it is in progress

**Review by zacisaacson — APPROVED** [review]

File: lib/team_stats.rb (hunk 1 above)
Comment (mcat56) [reply lib/team_stats.rb:96]: noted!

**Comment by freeheeling** [comment]
Full steam ahead!

**Pull request merged**